sitio/src/routes/chequear/+page.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+  import { dayjs } from "$lib/consts";
+  import { likedTweets } from "../../schema";
+  import Footer from "../Footer.svelte"
+
+  export let data: { found: [typeof likedTweets], error: string | null };
+  let test = $page.url.searchParams.get("test");
+
+  export let fecha = data.found?.length ? dayjs(data.found[0].firstSeenAt + "").format("YYYY-MM-DD") : ""
+  export let sinChequear = data.found === null // si no vino info está para chequear
```

Comment:
```suggestion
  $: sinChequear = data.found === null // si no vino info está para chequear
```

estas dos son variables reactivas a partir de `data`, asi que tienen que ser declaradas con `$: `. `export let` es para propiedades que vienen de afuera ("props")